schemas/user.py

```
@@ -0,0 +1,61 @@
+
+class UserUpdate(BaseModel):
+    nickname: str | None = Field(None,min_length=2,max_length=20,pattern=r"^[가-힣a-zA-Z0-9]+$")
+
+class UserResponse(UserBase):
+    id : str
+    profileImage: str | None
+    createdAt: datetime
+
+class Token(BaseModel):
+    access_token: str
+    token_type : str = "bearer"
```

Ruff (0.14.11)

Possible hardcoded password assigned to: "token_type"

(S105)